src/sdk.test.js

```diff
@@ -56,28 +57,44 @@ const createSdk = (config = {}) => {
 };
 
 describe('new SharetribeSdk', () => {
+  const validSdkConfig = {
+    clientId: '08ec69f6-d37e-414d-83eb-324e94afddf0',
+    baseUrl: 'https://api-base-url.example',
+    assetCdnBaseUrl: 'https://asset-cdn-base-url.example',
+  };
+
   it('validates presence of clientId', () => {
-    expect(() => new SharetribeSdk()).toThrowError('clientId must be provided');
+    expect(() => new SharetribeSdk(_.omit(validSdkConfig, ['clientId']))).toThrowError(
+      'clientId must be provided'
+    );
   });
 
-  it('validates presence of baseUrl', () => {
-    expect(
-      () =>
-        new SharetribeSdk({
-          clientId: '08ec69f6-d37e-414d-83eb-324e94afddf0',
-          baseUrl: null,
-        })
-    ).toThrowError('baseUrl must be provided');
+  it('validates that baseUrl is not explicitely set to null', () => {
+    // This test validates that baseUrl is not explicitely set to null. If
+    // baseUrl is missing, default baseUrl is used but if baseUrl is set to
+    // null, that's an error case we want to catch.
+    expect(() => new SharetribeSdk({ ...validSdkConfig, baseUrl: null })).toThrowError(
+      'baseUrl must be provided'
+    );
+  });
+
+  it('validates that assetCdnBaseUrl is not explicitely set to null', () => {
+    // This test validates that assetCdnBaseUrl is not explicitely set to null.
+    // If assetCdnBaseUrl is missing, default assetCdnBaseUrl is used but if
+    // assetCdnBaseUrl is set to null, that's an error case we want to catch.
+    expect(() => new SharetribeSdk({ ...validSdkConfig, assetCdnBaseUrl: null })).toThrowError(
+      'assetCdnBaseUrl must be provided'
+    );
   });
 
   it('uses default baseUrl, if none is set', () => {
     const adapter = createAdapter((config, resolve) => {
-      // Fake adapter that echoes the URL
+      // Fake adapter that echoes the URL that was used in the request
       resolve({ data: { baseURL: config.baseURL } });
     });
 
     const sdk = new SharetribeSdk({
-      clientId: '08ec69f6-d37e-414d-83eb-324e94afddf0',
+      ..._.omit(validSdkConfig, 'baseUrl'),
```

Comment:
This is a nice oneliner, but if there's a need to remove lodash from build, it's also possible to do something like:

```js
const arr = { a: 1, b: 2, c: 3 }
const { b, ...restWithouthB } = arr;
console.log(restWithouthB);
// "{a: 1, c: 3}"
```